Source/WTF/wtf/PlatformEnable.h

```diff
@@ -890,7 +890,8 @@
 #endif
 #endif
 
-#if !defined(ENABLE_PDFJS) && (PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(WPE))
+// FIXME: Reenable PDFJS by default for Cocoa (https://bugs.webkit.org/show_bug.cgi?id=235981).
+#if !defined(ENABLE_PDFJS) && ( /* PLATFORM(COCOA) || */ PLATFORM(GTK) || PLATFORM(WPE))
 #define ENABLE_PDFJS 1
 #endif
```

Comment:
One downside to moving this preprocessor macro logic to xcconfigs is that we don't have a single "Features" xcconfig that is used by every target. So, when building with `--pdfjs`, `ENABLE(PDFJS)` will be true in WebCore, WebKit, and WebKitLegacy, because those projects recognize the build setting. But it'll be false everywhere else.